bundle-workflow/src/run_integ_test.py

```diff
@@ -29,13 +29,16 @@
 def parse_arguments():
```

Comment:
I do see these args as common for all test suites we should unify them eventually as TestArgs. 
Doesn't have to be part of this PR.

Reply:
That's the plan. I avoided refactoring at this point as Bwc uses TestArgs with different arguments. After this is merged, and once bwc is refactored, this logic can be moved to TestArgs